liblouis/metadata.c

```diff
@@ -155,13 +158,14 @@ list_size(List *list) {
  * Convert a list into a NULL terminated array.
  */
 static void **
-list_toArray(List *list, void *(*dup)(void *)) {
+list_toArray(List *list, int deepCopy) {
```

Comment:
What is the `deepcopy` argument for? It never seems to be set to `true`. The whole thing seems a bit brittle: You have to set `deepcopy` and `list->dup` to make sure the list is copied. Do we really need the `deepcopy` argument?

Reply:
Yes maybe I ain't, but this doesn't add any complexity.